src/lib/dialog/dialog-directives.ts

```diff
@@ -0,0 +1,49 @@
+import {Directive} from '@angular/core';
+import {MdDialog} from './dialog';
+
+
+/**
+ * Button that will close the current dialog.
+ */
+@Directive({
+  selector: 'button[md-dialog-close]',
+  host: {
+    '(click)': 'dialog.closeTop()'
+  }
+})
+export class MdDialogClose {
+  constructor(public dialog: MdDialog) { }
+}
+
+/**
+ * Title of a dialog element. Stays fixed to the top of the dialog when scrolling.
+ */
+@Directive({
+  selector: '[md-dialog-title]',
+  host: {
+    role: 'heading'
```

Comment:
I added it in case the attribute was added to something like a `div`. I'll scope the selector to the proper nodes.